lib/Conversion/TorchToLinalg/DataMovement.cpp

```diff
@@ -770,6 +771,8 @@ class ConvertAtenViewOp : public OpConversionPattern<AtenViewOp> {
     SmallVector<int64_t> yIndicesResult({0});
     size_t nextXIndex = 1;
     size_t nextYIndex = 1;
+    if (xTotalSize == kUnknownSize)
```

Comment:
nit: can you move this to before the vectors `xIndicesResult` and `yIndicesResult` are created, to avoid unnecessary computation? Also, I think we should make the same check for `yTotalSize` and return `failure` if it is unknown at the start.

Reply:
done